eng/Directory.Build.Common.targets

```diff
@@ -29,7 +29,7 @@
   <!-- Validates that there is a release note entry for the current package version -->
   <Target Name="ValidateReleaseNotes" BeforeTargets="SetPackageProjectUrlandReleaseNotes" Condition="'$(IsShippingLibrary)' == 'true'" >
     <PropertyGroup>
-      <ValidateReleaseNotesScriptPath Condition=" '$(ValidateReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)common/scripts/Verify-ChangeLog.ps1</ValidateReleaseNotesScriptPath>
+      <ValidateReleaseNotesScriptPath Condition=" '$(ValidateReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)common\scripts\Verify-ChangeLog.ps1</ValidateReleaseNotesScriptPath>
```

Comment:
Why was this change necessary? We need to keep the forward slashes to ensure we work cross-platform.

Reply:
Reverted